app/code/community/Reverb/ReverbSync/Model/Observer.php

```diff
@@ -7,11 +7,24 @@ class Reverb_ReverbSync_Model_Observer
     //function to create the product in reverb
     public function productSave($observer)
     {
-        $productSyncHelper = Mage::helper('ReverbSync/sync_product');
-        $product_id = $observer->getProduct()->getId();
-        try
-        {
-            $listingWrapper = $productSyncHelper->executeIndividualProductDataSync($product_id);
+      try
+      {
+          $product = $observer->getProduct();
+          $product_id = $product->getId();
+
+          $syncToReverb = $product->getData('rev_sync');
+          if (is_null($syncToReverb)) {
+              // TODO: This will potentially generate one SQL query per product saved.  Revamp this to queue the check and process in async batches.
+              $syncToReverb = Mage::getResourceModel('catalog/product')->getAttributeRawValue($product_id, 'rev_sync');
```

Comment:
I think on the business level I'm in agreement that we should not throw warnings here. People turning off reverb sync are doing that on purpose. So if you guys both agree this is the right technical thing to do I'm good to merge.